examples/compute-management/02-VirtualMachineCreation.sh

```diff
@@ -3,15 +3,15 @@ set -e
 printf "\n=== Managing Virtual Machine Creation in Azure Compute ===\n"
 
 printf "\n1. Creating a new resource group: %s and location: %s.\n" "$groupName" "$location"
-az group create -n "$groupName" --location "$location"
+az resourcemanager group create -n "$groupName" --location "$location"
 
 printf "\n2. Creating a new storage account '%s' in type '%s'.\n" "$storageAccountName" "$storageAccountType"
 az storage account new --resourcegroupname "$groupName" --name "$storageAccountName" --location "$location" --type "$storageAccountType"
 
 printf "\n3. Create virtual network.\n"
-result=`az virtual network new --resourcegroupname "$groupName" --name test --location "$location" --addressprefix "[\"10.0.0.0/16\"]" --subnet "[{\"Name\":\"test\",\"AddressPrefix\":\"10.0.0.0/24\"}]" --force`
+result=`az network vnet new --resourcegroupname "$groupName" --name test --location "$location" --addressprefix "[\"10.0.0.0/16\"]" --subnet "[{\"Name\":\"test\",\"AddressPrefix\":\"10.0.0.0/24\"}]" --force`
```

Comment:
Are we realy going to confuse network vnets with other kinds of vnets?  I would think vnet would be a top level noun